mobile/cmp/grouping/GroupingChooser.scss

```diff
@@ -16,18 +16,23 @@
     }
   }
 
+  &__editor {
+    .xh-panel__content {
+      padding: var(--xh-pad-px);
+    }
+  }
+
   &__row {
     display: flex;
     align-items: center;
     background-color: var(--xh-bg);
-    height: 35px;
+    height: 45px;
```

Comment:
Now that we have the full screen on mobile, control rows felt less cramped with more height